docs/support/tutorials/hpc-quick.md

```diff
@@ -0,0 +1,145 @@
+# Getting started with supercomputing at CSC
+
+You have signed up for your CSC account and first project, and are now ready to
+scale up your computing! This page provides guidance for getting started with
+using our HPC resources. It is recommended that new users complete the
+[CSC Computing Environment course](https://csc-training.github.io/csc-env-eff/),
+which provides a more in-depth introduction to using CSC services.
+
+## Available systems
+
+### Puhti
+
+New users are recommended to start working on the
+[Puhti supercomputer](../../computing/systems-puhti.md).
+Compared to Mahti, it has much more pre-installed software, more GPU nodes, and
+typically more available memory per CPU. Additionally, some Puhti CPU nodes have
+fast local NVMe storage, which is reserved only for GPU nodes on Mahti.
+
+### Mahti
+
+If you know that your computations are highly parallelizable, you should
+consider running them on the
+[Mahti supercomputer](../../computing/systems-mahti.md).
+Compared to Puhti, Mahti has many more CPU nodes and cores per node. Mahti is
+intended for computations that are able to effectively utilize at least an
+entire CPU node.
+
+Additionally, while Mahti has fewer GPU nodes than Puhti, the A100 GPUs on
+Mahti are considerably more powerful than the V100 GPUs on Puhti, which
+makes Mahti also suitable for machine learning applications.
+
+### LUMI
+
+LUMI is one of the fastest supercomputers in the world. It is intended primarily
+for running computations that benefit from the large amount of high-performance
+GPUs in its LUMI-G hardware partition. Whereas the GPUs on Puhti and Mahti are
+manufactured by Nvidia, the LUMI GPUs are made by AMD, so make sure that your
+GPU applications are able to use AMD drivers. LUMI has
```

Comment:
```suggestion
GPU applications have been ported to run on AMD GPUs. LUMI has
```

Reply:
Hmm, but not all applications have to be ported, they might natively support AMD GPU? 
I think "are able to run on AMD GPUs" fits better here.

Reply:
Fair point :+1: